tcms/core/tests/test_admin.py

```diff
@@ -116,14 +118,13 @@ def test_delete_another_user(self):
 
     def test_delete_myself(self):
         self.client.login(  # nosec:B106:hardcoded_password_funcarg
-            username=self.tester.username,
+            username=self.superuser.username,
```

Comment:
Each user should be able to delete themselves so this change is not needed

Reply:
However superuser should be able to delete anyone so that is another test method which is missing